socorro/external/es/crashstorage.py

```diff
@@ -441,7 +441,7 @@ class ESCrashStorageRedactedJsonDump(ESCrashStorageRedactedSave):
     """
     required_config = Namespace()
     required_config.add_option(
-        name="json_dump_whitelist_keys",
+        name="json_dump_allowlist_keys",
```

Comment:
This is configuration, but we don't define it anywhere--we use the defaults from here.